api/src/main/java/no/nav/sbl/dialogarena/modiabrukerdialog/api/domain/saker/Sak.java

```diff
@@ -35,7 +35,6 @@
     public static final String BIDRAG_MARKOR = "BID-HACK";
 
     public static final List<String> GYLDIGE_FAGSYSTEM_FOR_GENERELLE_SAKER = List.of(FAGSYSTEM_FOR_OPPRETTELSE_AV_GENERELL_SAK, "");
-    public static final List<String> GODKJENTE_TEMA_FOR_GENERELL_SAK = List.of("AAP", "AGR", "BAR", "BIL", "DAG", "ENF", "ERS", "FEI", "FOR", "FOS", "FUL", "GEN", "GRA", "GRU", "HEL", "HJE", "IND", "KON", "KTR", "MED", "MOB", "OMS", "REH", "RVE", "RPO", "SAK", "SAP", "SER", "STO", "SUP", "SYK", "SYM", "TRK", "TRY", "TSR", "TSO", "UFM", "VEN", "YRA", "YRK", "FRI", TEMAKODE_OPPFOLGING);
```

Comment:
Litt usikker på om denne må fjernes, så eg lurer på om denne er basisen for kva som skal vises. Då det er nokre temakoder me tidlegare har ikkje gitt muligheiten for å opprette sak på eller vise at det finnes sak på (f.eks  saker knyttet til kontroll, dette er ikkje ønska at skal vere synlig).